<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:#31363B;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color:#31363B;
}
QTabWidget{
background-color:#31363B;
border: 0;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Logon</string>
     </attribute>
     <widget class="QLineEdit" name="usernameInput">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background-color:#31363B;
color:white;
border: 1px solid white;
border-radius:5px;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submitButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>170</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border: 1px solid white;
border-radius: 5px;
background-color: #31363B;
color: white;
}</string>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordInput">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background-color:#31363B;
color:white;
border: 1px solid white;
border-radius:5px;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Create Account</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>New Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="createPasswordInput">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background-color:#31363B;
color:white;
border: 1px solid white;
border-radius:5px;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>New Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="createUsernameInput">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background-color:#31363B;
color:white;
border: 1px solid white;
border-radius:5px;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submitAccountButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border: 1px solid white;
border-radius: 5px;
background-color: #31363B;
color: white;
}</string>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <zorder>createPasswordInput</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>createUsernameInput</zorder>
     <zorder>submitAccountButton</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>417</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>createUsernameInput</tabstop>
  <tabstop>createPasswordInput</tabstop>
  <tabstop>submitAccountButton</tabstop>
  <tabstop>usernameInput</tabstop>
  <tabstop>passwordInput</tabstop>
  <tabstop>submitButton</tabstop>
  <tabstop>tabs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
